app/service.py

```diff
@@ -117,9 +136,10 @@ def init_db():
 
 def check_content_type(content_type):
     """ Checks that the media type is correct """
-    if request.headers['Content-Type'] == content_type:
-        return
-    app.logger.error('Invalid Content-Type: %s', request.headers['Content-Type'])
+    if 'Content-Type' in request.headers:
```

Comment:
Why are you submit this change?

Reply:
I will add the test cases and the submit the complete code.